gradle.properties

```diff
@@ -15,10 +15,10 @@
 #
 
 group=org.sonatype.gradle.plugins
-version=2.4.2-SNAPSHOT
+version=2.5.0-SNAPSHOT
```

Comment:
Although there are no noticiable new features, I feel this is a big enough milestone specially for new contributors into making the development experience better, so a minor version bump felt right.